### DISCUSSION TITLE
Cors policy added in virtual service configuration not working as expected

### DISCUSSION THREAD

**Discussion by parshva-shah-dsw** [istio/istio#54429] [General]
@howardjohn We are following the below link to configure CORS for specific origin with exact url to be allowed "working.com". https://istio.io/latest/docs/reference/config/networking/virtual-service/#CorsPolicy

in both the cases of passing the origin header in the curl the request is going through 

curl -v -H "Origin: example.com" https://working.com/token
curl -v -H "Origin: working.com" https://working.com/token

also do we have any envoy filter example - https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/cors/v3/cors.proto#extensions-filters-http-cors-v3-cors

istio 1.23.0

**Comment by howardjohn** [comment]
"curl the request is going through"

CORS is a client side thing. It is **not** a server-side enforcement -- Istio (or any other server) will not reject a request for not matching the CORs. Instead, it will not return the various headers that tell a browser it is authorized to send the cross-origin requests.
